ipn/ipnlocal/serve_test.go

```diff
@@ -195,8 +195,9 @@ func TestServeHTTPProxy(t *testing.T) {
 		},
 		UserProfiles: map[tailcfg.UserID]tailcfg.UserProfile{
 			tailcfg.UserID(1): {
-				LoginName:   "someone@example.com",
-				DisplayName: "Some One",
+				LoginName:     "someone@example.com",
+				DisplayName:   "Some One",
+				ProfilePicURL: "someone-prof-pic.example.com",
```

Comment:
bad example as it's not a full URL. Do `https://example.com/photo.jpg` or something?